hudhook/src/hooks/dx11.rs

```diff
@@ -310,6 +207,24 @@ impl ImguiRenderer {
     }
 }
 
+impl crate::hooks::common::ImguiRendererCommon for ImguiRenderer {
```

Comment:
```suggestion
impl ImguiRendererCommon for ImguiRenderer {
```
Also here I'd rather just import the trait at the top.